www/htdocs/index.html

```diff
@@ -2619,6 +2626,331 @@
 ]
 
 ---
+---
+
+class: title
+
+# Metrics
+
+---
+
+## What are metrics?
+
+---
+
+## Which metrics will we collect?
+
+- node metrics (e.g. cpu, ram, disk space)
+
+- container metrics (e.g. memory used, processes, network traffic going in and out)
+
+- application metrics (e.g. TODO)
+
+---
+
+## Tools
+
+We will use three open source Go projects for metric collection, publishing, storing, and visualization:
+
+- Intel Snap: telemetry framework to collect, process, and publish metric data
+
+- InfluxDB: database
+
+- Grafana: graph visuals
+
+---
+
+## Snap 
+
+- [www.github.com/intelsdi-x/snap](www.github.com/intelsdi-x/snap)
+
+- Can collect, process, and publish metric data
+
+- Doesn’t store metrics
+
+- Works as a daemon
+
+- Offloads collecting, processing, and publishing to plugins
+
+- Have to configure it to use the plugins and collect the metrics you want
+
+---
+
+## InfluxDB
+
+- Since Snap doesn't have have a database, we need one
+
+- It's specifically for time series
+
+---
+
+## Grafana
+
+- Since neither Snap or InfluxDB can show graphs, we're using Grafana
+
+---
+
+## Snap and Global Scheduling
+
+---
+
+## Storage and UI Layers
+
+---
+
+## Getting Snap
+
+Get the latest Snap release
+.exercise[
+
+- Get Snap:
+  ```bash
+    docker run -v /opt:/opt centos sh -c "curl -sSL https://github.com/intelsdi-x/snap/releases/download/v0.15.0-beta/snap-v0.15.0-beta-linux-amd64.tar.gz | tar -C /opt -zxf-"
```

Comment:
I'd suggest to use `$SNAPVER` there too; it makes it super easy to switch to a different version with minimal edits.